src/social-network-provider/facebook.com/UI/injectPostDummy.tsx

```diff
@@ -0,0 +1,3 @@
+export function injectPostDummyFacebook() {
+    return () => {}
```

Comment:
So this feature doesn't support Facebook?

Reply:
For now, it's only for Twitter.